3rd-party/class.jetpack-amp-support.php

```diff
@@ -338,7 +338,14 @@ public static function render_sharing_html( $markup, $sharing_enabled ) {
 			$sharing_link   .= '></amp-social-share>';
 			$sharing_links[] = $sharing_link;
 		}
-		return preg_replace( '#(?<=<div class="sd-content">).+?(?=</div>)#s', implode( '', $sharing_links ), $markup );
+
+		// Wrap AMP sharing buttons in container.
+		$markup = preg_replace( '#(?<=<div class="sd-content">).+?(?=</div>)#s', implode( '', $sharing_links ), $markup );
+
+		// Remove any lingering share-end list items.
+		$markup = str_replace( '<li class="share-end"></li>', '', $markup );
```

Comment:
Ok, I don't understand why this is added to begin with if it's always empty.

Reply:
Yeah. Might be worth some investigation.

Reply:
It's been like that since the first release. I tracked it down to r27835-wpcom, but unfortunately that commit isn't too detailed:

> fixing alignment problems

I'd be tempted to not remove it for now. :)